specs/default/chef/site-cookbooks/symphony/recipes/default.rb

```diff
@@ -96,7 +96,7 @@
 
 
 # Install dependencies
-%w{ net-tools gettext unzip python}.each do |pkg|
+%w{ net-tools gettext unzip python2}.each do |pkg|
```

Comment:
Why python2 and not python3?

Reply:
we are installing python2 for symphony requestor plugin and python3 is installed as part of cluster-init script but can definitely move here too